R/R6.R

```diff
@@ -57,7 +61,7 @@ neo4j_api <- R6::R6Class(
         cat("(Wrong credentials or hostname)\n")
       }
     },
-    initialize = function(url, user, password) {
+    initialize = function(url, db, user, password, isV4) {
```

Comment:
previous `{neo4r}` implementation didn't have a `db` parameter, which is a breaking change. 

We should use default values here so that it doesn't break existing code :)

Reply:
It would also be nice to move `db` to the end of the parameter list.

Reply:
default `db` to `"neo4j"`